README.md

```diff
@@ -1,11 +1,20 @@
-# Consul [![CircleCI](https://circleci.com/gh/hashicorp/consul/tree/main.svg?style=svg)](https://circleci.com/gh/hashicorp/consul/tree/main) [![Discuss](https://img.shields.io/badge/discuss-consul-ca2171.svg?style=flat)](https://discuss.hashicorp.com/c/consul)
+# Consul
+
+<p>
+  <a href="https://consul.io" title="Consul website">
+    <img src="./website/public/img/logo-hashicorp.svg" alt="HashiCorp Consul logo" width="200px">
+  </a>
+</p>
```

Comment:
Super nitpicky but do we need the `<p>` tag here? Asking out of curiosity

It would be nice if we could just make a one-liner `<img>` since we have the website link below anyways

Reply:
Packer and Nomad do the same thing:
- https://raw.githubusercontent.com/hashicorp/packer/master/README.md
- https://raw.githubusercontent.com/hashicorp/nomad/master/README.md

I also found external references to this being the way to set a custom image width in Github flavored markdown. I don't know if there's a cleaner, more recent way to do this, but this is a standard way to do it. You'd never to it this way in HTML, but this isn't HTML :P